### PR TITLE
Accessibility Review

### PR DESCRIPTION
**Pull request by lucabenaventew**
# Accessibility Report
- Page titles - Updated (To see changes [click here](https://github.com/lucabenaventew/Portfolio/pull/18/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R11))
- Image text alternatives - Updated (To see changes [click here](https://github.com/lucabenaventew/Portfolio/pull/18/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L33-R45))
- Text headings - No issues found
- Color contrast - No issues found
- Resize - Updated (To see changes [click here](https://github.com/lucabenaventew/Portfolio/pull/18/files#diff-380b7b38760dd442e897eb0164c58f6a17da966ccaca6318017a468c163979b1R189-R529))
- Interaction - No issues found
- Moving content - No issues found
- Multimedia - No issues found
- The basic structure of the page - No issues found

### CODE REVIEW

**Review by OLIPLICHE — COMMENTED** [review]
Hi @lucabenaventew , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Please make sure, the node_modules folder should be in the .gitignore file

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <link rel="stylesheet" href="styles.css" />
     <link rel="stylesheet" href="grid.css" />
     <script src="https://kit.fontawesome.com/8bb8430e5d.js" crossorigin="anonymous"></script>
-    <title>Portfolio</title>
+    <title>Luca's Portfolio</title>
 </head>
 
 <body>
```
Comment (OLIPLICHE) [inline index.html:14]: - [ ] Please make sure that in the "Form section" (Form fields ), Every form control has a `label` associated with it using `label`, `for`, and `id`.

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @lucabenaventew , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Linter result is green ✔️
- Commit messages are meaningful✔️
- The pull request have a descriptive title telling you what changes are inside ✔️
- The pull request have a short summary describing introduced changes in more detail ✔️
- node_modules folder is in the .gitignore ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**You can also consider**:

- [ ] You did well in adding  "alt = ..." in the images (`alt text`). I suggest that you do the same for the image below because this image is important for understanding the content.


im1


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by lucabenaventew** [comment]
Hey @OLIPLICHE thank you for your review, I just wonder why you didnt told me this from the beggining, now I just have 1 attempt to send it, It has no sense for me.

**Comment by OLIPLICHE** [comment]
> Hey @OLIPLICHE thank you for your review, I just wonder why you didnt told me this from the beggining, now I just have 1 attempt to send it, It has no sense for me.

Hi @lucabenaventew, Thank you for reaching out. Please remember to use the same pull request as in your initial review. I reviewed your PR from the start because I did not meet any previous comments from the first review. I provide the link for you to learn more on [How to ask for code review](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/how_to_ask_for_a_code_review.md)

- [ ] PLEASE CONSIDER MY REQUIRED CHANGE AS OPTIONAL SINCE YOU HAVE ONLY ONE CHANCE LEFT. you can submit the same PR 👍

**Review by nicupop729 — APPROVED** [review]
<h1>Status: Approved 🥇 ✔</h1>
Hi @lucabenaventew,


**I strongly emphasize the importance of implementing the recommendations of the previous reviewer!**
**Please take the time needed to give a meaningful description of the alt attribute of the project image, as well as give labels to the inputs of your form!**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**